lib/evmone/analysis.hpp

```diff
@@ -162,10 +162,14 @@ struct instr_info
 
 struct block_info
 {
+    /// The total base gas cost of all instructions in the block.
     int64_t gas_cost = 0;
-    int stack_req = 0;     ///< The required stack height to execute the block.
-    int stack_max = 0;     ///< The relative maximum stack height in the block.
-    int stack_change = 0;  ///< The relative stack height change after the execution of the block.
+
+    /// The stack height required to execute the block.
+    int stack_req = 0;
+
+    /// The maximum stack height relative to the stack height at block start.
+    int stack_max = 0;
```

Comment:
Perhaps `stack_max_growth` represents this variable better?

Reply:
Better name. Renamed.